docs/source/algorithms/ILLSANSReduction-v1.rst

```diff
@@ -0,0 +1,239 @@
+    ILLSANSReduction(Run='010446.nxs', ProcessAs='Transmission',
+                     AbsorberInputWorkspace='Cdw', BeamInputWorkspace='Dbw',
+                     OutputWorkspace='wc_tr')
+    print('Water container transmission is {0:.3f}'.format(mtd['wc_tr'].readY(0)[0]))
+
+    # Water container
+    ILLSANSReduction(Run='010454.nxs', ProcessAs='Container',
+                     AbsorberInputWorkspace='Cdw', BeamInputWorkspace='Dbw',
+                     TransmissionInputWorkspace='wc_tr', OutputWorkspace='wc')
+
+    # Water transmission
+    ILLSANSReduction(Run='010445.nxs', ProcessAs='Transmission',
+                     AbsorberInputWorkspace='Cdw', BeamInputWorkspace='Dbw', OutputWorkspace='w_tr')
+    print('Water transmission is {0:.3f}'.format(mtd['w_tr'].readY(0)[0]))
+
+    # Water
+    ILLSANSReduction(Run='010453.nxs', ProcessAs='Reference',
+                     AbsorberInputWorkspace='Cdw', ContainerInputWorkspace='wc',
+                     BeamInputWorkspace='Dbw', TransmissionInputWorkspace='wc_tr',
+                     SensitivityOutputWorkspace='sens', OutputWorkspace='water')
+
+    # Process the dark current Cd/B4C for sample
+    ILLSANSReduction(Run='010462.nxs', ProcessAs='Absorber', OutputWorkspace='Cd')
+
+    # Process the empty beam for sample
+    ILLSANSReduction(Run='010413.nxs', ProcessAs='Beam', AbsorberInputWorkspace='Cd', OutputWorkspace='Db')
+
+    # Sample container transmission
+    ILLSANSReduction(Run='010444.nxs', ProcessAs='Transmission',
+                     AbsorberInputWorkspace='Cd', BeamInputWorkspace='Dbw', OutputWorkspace='sc_tr')
+    print('Sample container transmission is {0:.3f}'.format(mtd['sc_tr'].readY(0)[0]))
+
+    # Sample container
+    ILLSANSReduction(Run='010460.nxs', ProcessAs='Container',
+                     AbsorberInputWorkspace='Cd', BeamInputWorkspace='Db',
+                     TransmissionInputWorkspace='sc_tr', OutputWorkspace='sc')
+
+    # Sample transmission
+    ILLSANSReduction(Run='010585.nxs', ProcessAs='Transmission',
+                     AbsorberInputWorkspace='Cd', BeamInputWorkspace='Dbw', OutputWorkspace='s_tr')
+    print('Sample transmission is {0:.3f}'.format(mtd['s_tr'].readY(0)[0]))
+
+    # Sample
+    ILLSANSReduction(Run='010569.nxs', ProcessAs='Sample',
+                     AbsorberInputWorkspace='Cd', ContainerInputWorkspace='sc',
+                     BeamInputWorkspace='Db', SensitivityInputWorkspace='sens',
+                     TransmissionInputWorkspace='s_tr', OutputWorkspace='sample_flux')
+
+    # Convert to I(Q)
+    Q1DWeighted(InputWorkspace='sample_flux', NumberOfWedges=0, OutputBinning='0.003,0.001,0.027', OutputWorkspace='iq')
+
+    # Plot the output
+    import matplotlib.pyplot as plt
+    fig, ax = plt.subplots(subplot_kw={'projection':'mantid'})
+    plt.xscale('log')
+    plt.yscale('log')
+    ax.errorbar(mtd['iq'],'-rs')
+    ax.set_ylabel('I [cm-1]')
+    ax.legend()
+    fig.show()
+
+.. note::
```

Comment:
This looks like an important note. Why is it hidden at the very end of the documentation page?